kubecontrolplane/v1/types.go

```diff
@@ -35,9 +35,6 @@ type KubeAPIServerConfig struct {
 	// servicesNodePortRange is the range to use for assigning service public ports on a host.
 	ServicesNodePortRange string `json:"servicesNodePortRange"`
 
-	// consolePublicURL is an optional URL to provide a redirect from the kube-apiserver to the webconsole
```

Comment:
You can deprecate and don't use it, but we can't remove field from our GA APIs, just like that.

Reply:
sorry, I have interpreted this wrongly then:

```
// Compatibility level 4: No compatibility is provided, the API can change at any point for any reason.
```